### PR TITLE
fix(meshgateways): add display-name support to MeshGateways

### PR DESCRIPTION
**Pull request by johncowen**
See title

Closes https://github.com/kumahq/kuma-gui/issues/2969

### CODE REVIEW

File: features/mesh/builtin-gateways/Item.feature [hunk 1]
```diff
@@ -18,7 +18,7 @@ Feature: mesh / builtin-gateways / item
       KUMA_RULE_MATCHER_COUNT: 1
       KUMA_RULE_MATCH_COUNT: 2
       """
-    Given the URL "/meshes/default/meshgateways/gateway-1" responds with
+    Given the URL "/meshes/default/meshgateways/gateway-1.namespace" responds with
```
Comment (jijiechen) [inline features/mesh/builtin-gateways/Item.feature:21]: Can we make `namespace` something else? I think it's a variable, right?
Here it looks like a constant in the template.

**Comment by jijiechen** [comment]
Some nits, but the feature looks OK.

**Review by jijiechen — APPROVED** [review]

File: features/mesh/builtin-gateways/Item.feature (hunk 1 above)
Comment (johncowen) [reply features/mesh/builtin-gateways/Item.feature:21]: oh yeah 2 tics, will switch up

Comment (johncowen) [reply features/mesh/builtin-gateways/Item.feature]: I changed it to `kuma-system`, thanks!

**Pull request merged**